app/services/idv/steps/selfie_step.rb

```diff
@@ -17,11 +21,12 @@ def send_selfie_request
         selfie_response = DocAuth::Client.client.post_selfie(
           instance_id: instance_id, image: image.read,
         )
+
         if selfie_response.success?
           handle_successful_selfie_match
-        else
-          handle_selfie_step_failure(selfie_response)
         end
```

Comment:
Rubocop got mad at this so I patched it in 9fe6ce442bbc13b7f4775eecf1761626463cef50

Reply:
> Rubocop got mad at this so I patched it in [9fe6ce4](https://github.com/18F/identity-idp/commit/9fe6ce442bbc13b7f4775eecf1761626463cef50)

Thanks!